src/preferences/configobject.cpp

```diff
@@ -270,6 +283,24 @@ void ConfigObject<ConfigValue>::setValue(
     set(key, ConfigValue(QString::number(value)));
 }
 
+template<>
+template<>
+void ConfigObject<ConfigValue>::setValue(
+        const ConfigKey& key, const mixxx::RgbColor::optional_t& value) {
+    if (!value) {
+        set(key, ConfigValue(""));
+        return;
+    }
+    set(key, ConfigValue(mixxx::toQColor(value.value()).name(QColor::NameFormat::HexArgb)));
```

Comment:
That's what toQString() is suppossed to do, but without the alpha channel. If the function name is too generic we could adjust it as needed.